torch/testing/_internal/common_device_type.py

```diff
@@ -1314,3 +1311,7 @@ def skipMeta(fn):
 
 def skipXLA(fn):
     return skipXLAIf(True, "Marked as skipped for XLA")(fn)
+
+
+def get_all_device_types() -> List[str]:
```

Comment:
We should add a note / file an issue that there are other device types than cpu and cuda (like XLA and MPS) and this should enumerate using the available device type test base classes